docs/src/school.md

```diff
@@ -0,0 +1,2 @@
+# DFTK School 2022
+For more details and registration info see the [school's website](https://school2022.dftk.org).
```

Comment:
so just remove it?

Reply:
Ah that will just redirect. No-one will ever see this.

Reply:
it's just because documenter does not allow you to put links into the sidebar.

Reply:
Oh strange, ok